apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -160,12 +166,13 @@ export default class EnrollmentsPanel extends React.Component {
       });
     } else {
       this.setState(state => {
-        state.selectedEnrollments.push({
+        const selectedEnrollments = state.selectedEnrollments.concat({
```

Comment:
Wow, I've never seen way of modifying state!

Reply:
yes, it is a bit overkill! I think this way only exists for when you need to do something that's more complicated than what's being done here, but the change I'm making didn't seem big enough to be worth reimplementing it.